tests/lib/rules/jsx-no-target-blank.js

```diff
@@ -27,7 +27,8 @@ ruleTester.run('jsx-no-target-blank', rule, {
   valid: [
     {code: '<a href="foobar"></a>', parserOptions: parserOptions},
     {code: '<a randomTag></a>', parserOptions: parserOptions},
-    {code: '<a href="foobar" target="_blank" rel="noopener noreferrer"></a>', parserOptions: parserOptions}
+    {code: '<a href="foobar" target="_blank" rel="noopener noreferrer"></a>', parserOptions: parserOptions},
+    {code: '<a target="_blank" {...spreadProps} rel="noopener noreferrer"></a>', parserOptions: parserOptions}
```

Comment:
It only matters because we break out of the `for` loop as soon as we find the `rel` attribute we're looking for.  I could add another test case, but it would pass on the old code, so doesn't illustrate the problem.  I didn't think it worth it at the time, but if you think it is, I'm happy to add it.